tools/editor/plugins/canvas_item_editor_plugin.h

```diff
@@ -35,13 +35,19 @@
 #include "scene/gui/panel_container.h"
 #include "scene/gui/box_container.h"
 #include "scene/2d/canvas_item.h"
-
+#include "scene/2d/light_2d.h"
```

Comment:
I might be wrong as I'm a C++ newbie, but wouldn't it be better practice to add those includes in the `.cpp` file where they're used, instead of the `.h`? Or is it necessary to have them here?